### ISSUE TITLE
How to use your pretrained model to generate registered images?

### ISSUE THREAD

**Issue by Stella-123-spu** [weixy17/IKCG#2]
Excellent work and impressive performance on ACROBAT dataset! I’m wondering if there are any instructions or guidelines on how to use your model to generate registered pairs of images. Thanks!

**Comment by weixy17** [comment]
Thanks for your interest in IKCG. We have provided the training pipeline and Command Lines in README.md. 


You can use 
TMI_rebuttle_main_baseline512.py or any other TMI_rebuttle_main_*.py files to load the pretrained model do the registration.

If you just want to get the registered images or optical flow fileds without training process, delete 'loss.backward()' and other related codes in  /network/pipeline.py.

/reader/ACROBAT.py contains the function LoadACROBAT_baseline512 to load the images to be registered. Image Preprocessing steps are introduced in our paper.


---- Replied Message ----
| From | Sitong ***@***.***> |
| Date | 4/3/2025 05:57 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [weixy17/IKCG] How to use your pretrained model to generate registered images? (Issue #2) |

Excellent work and impressive performance on ACROBAT dataset! I’m wondering if there are any instructions or guidelines on how to use your model to generate registered pairs of images. Thanks!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Stella-123-spu created an issue (weixy17/IKCG#2)

Excellent work and impressive performance on ACROBAT dataset! I’m wondering if there are any instructions or guidelines on how to use your model to generate registered pairs of images. Thanks!

**Comment by weixy17** [comment]
The model is provided at https://github.com/weixy17/IKCG/tree/main/ACROBAT/weights 


To load the pretrained model, just run 
python TMI_rebuttle_main_*.py MaskFlownet_S.yaml --dataset_cfg ACROBAT.yaml -g 0 -c 994Jan31-1235 --clear_steps --weight 1000 --batch 1






| |
魏星月
|
|
***@***.***
|
---- Replied Message ----
| From | ***@***.***> |
| Date | 8/14/2025 17:38 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [weixy17/IKCG] How to use your pretrained model to generate registered images? (Issue #2) |
labanzhao left a comment (weixy17/IKCG#2)

Please give me the Pytorch and mxnet vision, which is you use. I excel the step2 is error.
